pkg/ui/src/Header/Header.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+	import Logo from './Logo.svelte';
+	export let title = 'Stock';
+	const links = [
+		{
+			label: 'Stock',
+			href: '/inventory/stock'
+		},
+		{
+			label: 'Inbound',
+			href: '/inventory/inbound'
+		},
+		{
+			label: 'Outbound',
+			href: '/inventory/outbound'
+		}
+	];
+	export let currentUrl = 'stock';
+</script>
+
+<div class="bg-gray-900 p-10 pt-5 ">
+	<div class="border-b border-sky-900 pb-5 flex sm:items-center ">
+		<Logo />
+
+		<div class="pl-4">
```

Comment:
I think setting the `Logo` in a `<div class="mr-4">` makes more sense that setting the following nav into additional container and applying `ml-4`, let alone `pl-4`. 

The flow always goes from top to bottom and from left to right, so it's the easiest to organise the layout (html/css) in that way.